pyatlan/model/enums.py

```diff
@@ -2060,3 +2060,12 @@ class SourceCostUnitType(str, Enum):
     CREDITS = "Credits"
     BYTES = "bytes"
     SLOT_MS = "slot-ms"
+
+
+class jsdk_DataQualityType_RVUuI(str, Enum):
```

Comment:
This is a generated enum from integration tests — should revert to the last version of `enums.py` without this class.

Reply:
done